test/integration/patients/patient/archive.js

```diff
@@ -11,76 +21,101 @@ context('patient archive page', function() {
     cy
       .routesForPatientAction()
       .routePatient(fx => {
-        fx.data.id = '1';
-        fx.data.relationships.workspaces.data = [
-          {
-            id: '11111',
-            type: 'workspaces',
+        fx.data = getPatient({
+          id: '1',
+          relationships: {
+            workspaces: getRelationship(workspaceOne),
           },
-        ];
+        });
 
         return fx;
       })
       .routePatientActions(fx => {
-        fx.data = _.sample(fx.data, 3);
-        fx.data[0] = {
-          id: '1',
-          attributes: {
-            name: 'First In List',
-            details: null,
-            duration: 0,
-            due_date: null,
-            due_time: null,
-            updated_at: testTs(),
-          },
-          relationships: {
-            patient: { data: { id: '11111' } },
-            owner: {
-              data: {
-                id: '11111',
-                type: 'clinicians',
-              },
+        fx.data = [
+          getAction({
+            id: '1',
```

Comment:
one thing I tried in the recent tests that might be better.. I saved the resources that are used to test urls such that rather than setting an id to `1` and testing `visit('/foo/1')` I switched to ````visit(`/foo/${ resource.id }`)```

I think we could probably accomplish that here which will probably safer in the future.

Reply:
Why is that method the safer option?

Reply:
well the trend to move towards uuid v5 requires uuid ids